.travis.yml

```diff
@@ -22,7 +22,7 @@ before_install:
   - sudo rm -f /etc/boto.cfg
 env:
   global:
-    - CENTAUR_BRANCH=develop
+    - CENTAUR_BRANCH=rm_monitoring
```

Comment:
Yes -- I was passing in options through the inputs key in the Centaur test setup. Option was being ignored and hence no `monitoring.log` ever created.